src/cli.js

```diff
@@ -22,6 +23,11 @@ function cli(cwd) {
     },
     argv => handler(argv)
   );
+  yargs.command({
```

Comment:
This is not super compatible with the plugin-serverless atm. I'm fine with it being a separate command, we should then just have it part of the plugin somehow.

Reply:
I considered this just being a flag (`create-twilio-function --list-templates`) but I think that would mean I'd lose the default command with a required positional argument and the benefits that brings to its default, outside of plugin, use.

How can we improve this so that the plugin can best use it?